app/models/label/LabelTable.scala

```diff
@@ -81,6 +83,15 @@ object LabelTable {
   val neighborhoods = regions.filter(_.deleted === false).filter(_.regionTypeId === 2)
 
 
+  val anonId = "97760883-8ef0-4309-9a5e-0c086ef27573"
```

Comment:
Flagging it here: For future, when you update the role table, you should write the query that retrieves the anonymous user id and not hard code it.